docs/portal.properties-Reference.md

```diff
@@ -66,6 +66,18 @@ app.name should be set to the name of the portal instance referenced in the "AUT
 The default view in OncoPrint ("patient" or "sample") can be set with the following option. The default is "patient".
 ```
 oncoprint.defaultview=sample
+```
+## Custom annotation of driver and passenger mutations
```

Comment:
I think it is better to include them in the "Oncoprint" section (look also at the properties, they start with `oncoprint`.